diagnostics/model_test/verify_model.py

```diff
@@ -2,10 +2,38 @@
 import argparse
 import numpy as np
 import ngraph
+from google.protobuf import text_format
 import json
 import os
 
 
+def createFolder(directory):
+    try:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+    except OSError:
+        print('Error: Creating directory. ' + directory)
+
+
+def set_os_env(select_device):
+    if select_device == 'CPU':
+        # run on TF only
+        ngraph.disable()
+        os.environ['NGRAPH_TF_BACKEND'] = 'CPU'
+    else:
+        if not ngraph.is_enabled():
+            ngraph.enable()
+
+        back_end = select_device.split("NGRAPH_")
+        os.environ['NGRAPH_TF_BACKEND'] = back_end[1]
+
+        if quantized_mode == 'QUANTIZED':
+            os.environ['NNPI_CONVERT_FLOAT_TO_QUANT_GRAPH'] = '1'
+        else:
+            # float mode
+            os.environ['NNPI_CONVERT_FLOAT_TO_QUANT_GRAPH'] = '0'
+
```

Comment:
```suggestion
os.environ['NNPI_CONVERT_FLOAT_TO_QUANT_GRAPH'] = '1' if quantized_mode == 'QUANTIZED' else '0'
```

Also why have quantized_mode be a string. if its a number then we can simply do:
`os.environ['NNPI_CONVERT_FLOAT_TO_QUANT_GRAPH'] = str(quantized_mode)`

Reply:
You're right. Let me update json file also to get quantized mode as a number.